apps/dashboard-service/src/dashboard/total-jobs-created.controller.ts

```diff
@@ -0,0 +1,75 @@
+import { Get, Query, Controller } from "@nestjs/common";
+import { ApiOperation } from "@nestjs/swagger";
+import { JsonApiResponse } from "@terramatch-microservices/common/decorators";
+import { CacheService } from "./dto/cache.service";
+import { buildJsonApi, getStableRequestQuery } from "@terramatch-microservices/common/util/json-api-builder";
+import { DelayedJobDto } from "@terramatch-microservices/common/dto/delayed-job.dto";
+import { NoBearerAuth } from "@terramatch-microservices/common/guards";
+import { TotalJobsCreatedDto } from "./dto/total-jobs-created.dto";
+import { TotalJobsCreatedService } from "./total-jobs-created.service";
+import { DashboardQueryDto } from "./dto/dashboard-query.dto";
+
+@Controller("dashboard/v3/totalJobsCreated")
+export class TotalJobsCreatedController {
+  constructor(
+    private readonly cacheService: CacheService,
+    private readonly jobsCreatedService: TotalJobsCreatedService
+  ) {}
+
+  @Get()
+  @NoBearerAuth
+  @JsonApiResponse([TotalJobsCreatedDto, DelayedJobDto])
+  @ApiOperation({ operationId: "getTotalJobsCreated", summary: "Get total jobs created" })
+  async getTotalJobsCreated(@Query() query: DashboardQueryDto) {
+    const cacheKey = `dashboard:jobs-created|${this.cacheService.getCacheKeyFromQuery(query)}`;
+    let cachedData = await this.cacheService.get(cacheKey);
+    if (cachedData == null) {
+      cachedData = await this.jobsCreatedService.getTotals(query);
+      await this.cacheService.set(cacheKey, JSON.stringify(cachedData));
+    }
```

Comment:
The cache service got an update that could be used here to improve the code a bit:
```
const data = await this.cacheService.get(cacheKey, () => this.jobsCreatedService.getTotals(query));
```
The new optional factory parameter will only be called if the cache get is a miss, and will stash the returned value before returning it.